dask/dataframe/tests/test_dataframe.py

```diff
@@ -36,7 +36,7 @@ def eq(a, b):
        ('x', 2): pd.DataFrame({'a': [7, 8, 9], 'b': [0, 0, 0]},
                               index=[9, 9, 9])}
 d = dd.DataFrame(dsk, 'x', ['a', 'b'], [0, 4, 9, 9])
-full = d.compute()
+full = d.compute(get=get_sync)
```

Comment:
@mrocklin : without these two as `get_sync`, the test suite locks up in py.test after the changes in a62c666ab72b1b983b05dbe3e9a72c6d18c49add. I can manually import the test suite and run all the tests, and all tests in functions still use the default scheduler, but for some reason these two top-level calls lock up if not using `get_sync` only in py.test. `pdb` has been of little help here, I'm really not sure what's going on.